### ISSUE TITLE
Expanding the Azure Storage Explorer Emulator Bl;ob containers node errored

### ISSUE THREAD

**Issue by bennyHot** [microsoft/AzureStorageExplorer#4391]
Unable to retrieve child resources.

Details:
{
  "name": "FetchError",
  "message": "request to http://127.0.0.1:10000/devstoreaccount1?include=metadata&comp=list failed, reason: connect ECONNREFUSED 127.0.0.1:10000",
  "stack": "FetchError: request to http://127.0.0.1:10000/devstoreaccount1?include=metadata&comp=list failed, reason: connect ECONNREFUSED 127.0.0.1:10000\n    at ClientRequest.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\node-fetch\\lib\\index.js:1461:11)\n    at ClientRequest.emit (events.js:310:20)\n    at Socket.socketErrorListener (_http_client.js:426:9)\n    at Socket.emit (events.js:310:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)",
  "type": "system",
  "errno": "ECONNREFUSED",
  "code": "ECONNREFUSED"
}

**Comment by jinglouMSFT** [comment]
Do you have the emulator started?

**Comment by bennyHot** [comment]
Thanks for the quick response.
I found that the storage explorer does not install or start the emulator. Read that azurite is the way going forward.

So I installed azurite.
I started azurite but and showed ip in Azure Storage Explorer and I was able to create containers but Visual Studio Azure function had a problem connecting to it.

So I installed storage-emulator and started it.
https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#storage-emulator-command-line-tool-reference

So I have a working emulator accessible from code.

Thanks,
Ben



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
